target/linux/x86/base-files/lib/preinit/01_sysinfo

```diff
@@ -1,3 +1,12 @@
+sanitize_name_x86() {
+	sed -e '
+		y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/;
+		s/[^a-z0-9_-]\+/-/g;
+		s/^-//;
+		s/-$//;
+	' $@
```

Comment:
Tiny nit-pick, can you please quote? `"$@"`

Reply:
Good catch. Thanks.